.github/workflows/ci.yml

```
@@ -29,6 +29,8 @@ jobs:
         run: nix develop --store ~/.ci-store --command check-nixpkgs-fmt
       - name: Check EditorConfig conformance
         run: nix develop --store ~/.ci-store --command check-editorconfig
+      - name: Check Semantic Versioning is respected
+        run: nix develop --store ~/.ci-store --command check-semver
```

Once you want to disable this (after you're done testing it does what you want), I think you can add this to do so:
```suggestion
        if: false # disabled until we cut a real release
        run: nix develop --store ~/.ci-store --command check-semver
```